backend/src/crd/controller/scheduledworkflow/util/scheduled_workflow_test.go

```diff
@@ -287,11 +288,97 @@ func TestScheduledWorkflow_GetNextScheduledEpoch_OneTimeRun(t *testing.T) {
 		},
 	})
 	nextScheduledEpoch, mustRunNow = schedule.GetNextScheduledEpoch(
-		int64(1) /* active workflow count */, nowEpoch)
+		int64(1) /* active workflow count */, nowEpoch, time.Location{})
 	assert.Equal(t, false, mustRunNow)
 	assert.Equal(t, creationTimestamp.Unix(), nextScheduledEpoch)
 }
 
+func TestScheduledWorkflow_GetNextScheduledEpoch_CronScheduleTimeZone(t *testing.T) {
+	locationString := "America/Los_Angeles"
+	viper.Set(TimeZone, locationString)
+	defer viper.Set(TimeZone, "")
+
+	location, err := time.LoadLocation(locationString)
+	assert.Nil(t, err)
+	nowTime, err := time.Parse(time.RFC1123Z, "Mon, 03 Jan 2006 14:04:05 -0800")
+	assert.Nil(t, err)
+	nowEpoch := nowTime.Unix()
+
+	creationTime, err := time.Parse(time.RFC1123Z, "Mon, 01 Jan 2006 16:04:05 -0800")
+	assert.Nil(t, err)
+	creationTimestamp := metav1.NewTime(creationTime)
+	//catchUp := false
+	schedule := NewScheduledWorkflow(&swfapi.ScheduledWorkflow{
+		ObjectMeta: metav1.ObjectMeta{
+			CreationTimestamp: creationTimestamp,
+		},
+		Spec: swfapi.ScheduledWorkflowSpec{
+			Enabled: true,
+			//NoCatchup:      &catchUp,
+			MaxConcurrency: commonutil.Int64Pointer(int64(10)),
+			Trigger: swfapi.Trigger{
+				CronSchedule: &swfapi.CronSchedule{
+					Cron: "* * 15 * * *",
```

Comment:
nit: can you add a comment explaining what it does for every Cron expresson?

Reply:
Sure, will do!